app/containers/markdown/new/Katex.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { KaTeX as KaTeXProps } from '@rocket.chat/message-parser';
+// eslint-disable-next-line import/no-unresolved
+import MathView, { MathText } from 'react-native-math-view';
+
+interface IKaTeXProps {
+	value: KaTeXProps['value'];
+}
+
+export const KaTeX = ({ value }: IKaTeXProps): React.ReactElement | null => <MathView math={value} />;
+
+export const InlineKaTeX = ({ value }: IKaTeXProps): React.ReactElement | null => (
+	<MathText value={`$${value}$$`} direction='ltr' />
```

Comment:
Is it going to work on `arabic` language, which forces RTL?

Reply:
Mathematics is a language, like javascript, so Arabs or Brazilians speak the same language.
As I remember there are some calculations that can be written in RTL but the default is worldwide.
An edge case so to speak, if someone "complains" we can think about it in the future.